lib/auth/windows/ldap.go

```diff
@@ -167,6 +181,47 @@ func (c *LDAPClient) Close() {
 	c.mu.Unlock()
 }
 
+func extractReferrals(err error) []string {
```

Comment:
I'd add some godocs to this one. 

Maybe even make it accept an LDAPError instead of a generic error since it can't operate on any other type of error.

Reply:
I've added some documentation and changed arg to `LDAPError`